index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <meta http-equiv="x-ua-compatible" content="ie=edge">
+    
+  <!-- build:css css/main.css -->
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  <!-- endbuild -->
+
+  <title>Dangelo Arrivillaga Portfolio</title>
+</head>
+
+<body>
+  <div class="contenedor">
+    <header>
+      <div class="container">
+        <div class="h-block">
+          <a class="h2-style">Portfolio</a>
+        </div>
+        <div class="h-block container fd-rr mr">
+          <div class="c-container">
+            <hr class="b-hamburger">
+            <hr class="b-hamburger">
+            <hr class="b-hamburger">
+          </div>
+        </div>
```

Comment:
- [ ]  Since it represents the navbar, I would suggest you wrap them inside a `nav` tag